README.md

```diff
@@ -420,6 +420,11 @@ Variable | Description | Default value
 `DISABLE_HTTPS` | Disable HTTPS, this can be useful if TLS connections are going to be handled outside of this setup | 1
 `ENABLE_HTTP_REDIRECT` | Redirects HTTP traffic to HTTPS | 1
 `LOG_LEVEL` | Controls which logs are output from prosody and associated modules | info
+`VIDEO_ASPECTRATIO` | Sets aspect ratio of client video stream | 16/9
+`VIDEO_HEIGHT_IDEAL` | Preferred resolution delivered by client browser | 720
+`VIDEO_HEIGHT_MAX` | Maximal resolution delivered by client browser | 720
+`VIDEO_HEIGHT_MIN` | Minimal resolution delivered by client browser | 240
```

Comment:
These change need to be applied to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md (cf: #601)

Reply:
By whom? Should I do a fork and PR there or is this just a general idea?

Reply:
The whole instruction part has been moved to the handbook and leaving only this part on README.md does actually not make sense.

> By whom?

If you will, please go ahead- if you will not, please remove that part from README.md from this PR and I will take care of the rest.

Reply:
OK. I removed it in https://github.com/HenriWahl/docker-jitsi-meet/commit/b748828184cec8ad91866678420fcd607ca2aedf